app/views/stations/dashboard.erb

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <h1>Stations Dashboard</h1>
+</div>
+
+<div class="container">
+  <ul class="list-group" id="dashboard">
+    <li class="list-group-item">Total count: <%= Station.count %></li>
```

Comment:
same here - we want to get this in the controller and pass it through via an instance variable. this is too much for the view to handle. as it stands, we're basically bypassing the controller and going straight to the model